firefox_ui_tests/update/manifest.ini

```diff
@@ -0,0 +1,5 @@
+[test_direct_update.py]
+direct = true
```

Comment:
I see now this is working basically as expected, but what does this give us beyond just having separate manifests?